cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -257,7 +257,7 @@ public void execute() {
         }
 
         // If project is empty
-        if (ProjectUtils.isProjectEmpty(project)) {
+        if (ProjectUtils.isProjectEmpty(project) && !ProjectUtils.areBuildToolsAvailable(project)) {
```

Comment:
Shall we delay the check instead? Moving the current check to CompileTask would fix the issue I believe.

Reply:
Changed the PR according to this comment